governance/polls/Add GUNI-DAI-USDC as a new Vault Type - September 6, 2021.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Add GUNI-DAI-USDC as a new Vault Type - September 6, 2021
```

Comment:
We should change this. It's too generic. This should be the ilk name, I think we want: GUNIV3DAIUSDC-A

Reply:
Should change it elsewhere as well.

Reply:
Done!